apps/condo/domains/common/utils/serverSchema/accessControl.js

```diff
@@ -0,0 +1,147 @@
+const { transformCRUDString } = require('@condo/domains/common/utils/serverSchema/accessUtils')
+const { GQL_LIST_SCHEMA_TYPE, GQL_CUSTOM_SCHEMA_TYPE, getSchemaCtx } = require('@core/keystone/schema')
+const { get } = require('lodash')
+const { parseFieldAccess, parseListAccess, parseCustomAccess } = require('@keystonejs/access-control')
+
+function getListPermissions (user, listKey) {
+    return transformCRUDString(get(user, ['permissions', 'lists', listKey, 'access'], ''))
+}
+
+function getFieldPermissions (user, listKey, field) {
+    return transformCRUDString(get(user, ['permissions', 'lists', listKey, 'fields', field], ''))
+}
+
+function getCustomPermissions (user, key) {
+    return get(user, ['permissions', 'custom', key], null)
+}
+
+function hasDefinedPermissions (user) {
+    return get(user, 'permissions') != null
+}
+
+function denyIfHasNoPermission (user, permission, originalAccessResult) {
+    if (originalAccessResult === false) return false
+    if (hasDefinedPermissions(user)) {
+        return permission === true
+    }
+    return originalAccessResult
+}
+
+function createStaticAccessWrapper (permissionsGetter, defaultAccessValue = true, useOperationIndexing = true) {
```

Comment:
why default access is true?

Reply:
Because function is used only in this file and there are no other calls, only with true argument

Reply:
I can delete default value but there is no difference